tests/test_profile_management.py

```diff
@@ -1006,4 +1006,102 @@ def test_rename_publications_from_api2(self, client, profile_management, test_cl
 Thanks,
 
 The OpenReview Team.
-'''
\ No newline at end of file
+'''
+
+    def test_merge_profiles(self, client, profile_management, helpers):
+
+        rachel_client = helpers.create_user('rachel@profile.org', 'Rachel', 'Last', alternates=[], institution='google.com')
+        profile = rachel_client.get_profile()
+
+        profile.content['homepage'] = 'https://google.com'
+        profile.content['names'].append({
+            'first': 'Rachel',
+            'middle': 'Alternate',
+            'last': 'Last'
+            })
+        rachel_client.post_profile(profile)
+        profile = rachel_client.get_profile(email_or_id='~Rachel_Last1')
+        assert len(profile.content['names']) == 2
+        assert 'username' in profile.content['names'][1]
+        assert profile.content['names'][1]['username'] == '~Rachel_Alternate_Last1'
+
+        helpers.create_user('rachel@gmail.com', 'Rachel', 'First', alternates=[], institution='google.com')
+
+        request_note = rachel_client.post_note(openreview.Note(
+            invitation='openreview.net/Support/-/Profile_Merge',
+            readers=['openreview.net/Support', '~Rachel_Last1'],
+            writers=['openreview.net/Support'],
+            signatures=['~Rachel_Last1'],
+            content={
+                'name': 'Rachel Last',
+                'left': '~Rachel_Last1',
+                'right': '~Rachel_First1',
+                'comment': 'please merge my profiles',
+                'status': 'Pending'
+            }
+
+        ))
+
+        helpers.await_queue()
+
+        messages = client.get_messages(to='rachel@profile.org', subject='Profile merge request has been received')
+        assert len(messages) == 1
+        assert messages[0]['content']['text'] == '''Hi Rachel Last,
+
+We have received your request to merge the following profiles: ~Rachel_Last1, ~Rachel_First1.
+
+We will evaluate your request and you will receive another email with the request status.
+
+Thanks,
+
+The OpenReview Team.
+'''
+
+        ## Accept the request
+        decision_note = client.post_note(openreview.Note(
```

Comment:
i think instead of just accept or reject, the direction of merge also need to be specified

Reply:
Maybe the direction should be for us to decide

Reply:
discussed this in the daily meeting.
the actual profile merge will be done by the moderator in profile compare page(same as how it's done currently)

the user moderation page will have a link to profile compare page and after the profile is merged the moderator will go back to moderation page to click approve to indicate the profile merge has been done.(or don't merge and reject)